components/tools/OmeroJava/test/integration/ModelMockFactory.java

```diff
@@ -86,16 +86,16 @@
     static int SIZE_T = 10;
 
     /** The dimension order for the pixels type. */
-    static String XYZCT = "XYZCT";
+    static String XYZCT = ome.model.enums.DimensionOrder.VALUE_XYZCT;
```

Comment:
These classes should already have access to constants like `omero.model.enums.DimensionOrderXYZCT`.